packages/openactive-integration-tests/test/templates/u-req.js

```diff
@@ -12,17 +12,20 @@ const { BOOKING_API_BASE } = global;
  * @param {UReqTemplateData} data
  */
 function createStandardUReq(data) {
-  return {
+  const req = {
     '@context': 'https://openactive.io/',
     '@type': 'Order',
-    orderedItem: [
-      {
-        '@type': 'OrderItem',
-        '@id': `${data.orderItemId}`,
-        orderItemStatus: 'https://openactive.io/CustomerCancelled',
-      },
-    ],
+    orderedItem: [],
   };
+  for (const orderItemId of data.orderItemIdArray) {
```

Comment:
simpler as a `.map(..)`?

Reply:
I actually disagree with this one as it's not returning an array